docs/install/osx.rst

```diff
@@ -1,42 +0,0 @@
-.. _osx_source:
```

Comment:
We also changed this document to get rid of the brew instructions, I'm not sure why it didn't show up. I'm not allowed to make a suggestion on deleted lines, but I'll make a PR to your branch with the changes.

Reply:
it probably show up but as we only see a diff between 0.7.0-rc branch and the last commit of this one, we don't see it.
basically, it had it, but got overwritten by the deletion...

Reply:
I replaced both the `linux_source.rst` and the `osx_source.rst` by the `source.rst`
(the argument was there is no point on explaining how to get the dependencies on OSX if we don't explain how to get them on linux)

Also the point was brew is only present on some of the system, forcing new users to use it, might messed up their system.